pkg/operation/common/utils.go

```diff
@@ -428,6 +428,19 @@ func DeleteLoggingStack(ctx context.Context, k8sClient client.Client, namespace
 	return nil
 }
 
+// GetLokiHvpaRecommendedResources  returns reccomended resources for Loki by the HVPA
+func GetLokiHvpaRecommendedResources(ctx context.Context, k8sClient client.Client) (*corev1.ResourceRequirements, error) {
```

Comment:
Can you implement a unit test for it? Also, maybe make it more general, i.e., change its signatur to
```golang
GetResourcesForContainerInStatefulSet(context.Context, client.Client, key client.ObjectKey, containerName string) (*corev1.ResourceRequirements, error)
```

Reply:
> Can you implement a unit test for it?

I am not sure how exactly can I mock the `k8sClient.Get(ctx, key, statefulSet)` function save in the given statefulSet, the wanted object

Reply:
There are plenty of examples in other unit tests that you can take a look at and get inspiration from :)

Reply:
done